doc: samples: suit: Testing Steps for recovery sample (#18859)
nrfconnect/sdk-nrf: samples/suit/recovery/README.rst

|test_sample|

#. |connect_kit|

> #. Corrupt the currently running main application. This can be done by overwriting the beggining of the application partition.

Reviewer: ```suggestion
#. Corrupt the currently running main application.
   This can be done by overwriting the beginning of the application partition:
```

.. code-block:: console

   nrfutil device x-write --address 0xE0A5000 --value 0xFFFF --serial-port COM7

> #. Reset the device to enter the recovery path.

Reviewer: ```suggestion
#. Reset the device to enter the recovery path:
```

.. code-block:: console

   nrfutil device reset --serial-port COM7

> #. Open the nRF Device Manager app on your mobile phone, and observe the device advertising itself now as "SUIT Recovery".

Reviewer: ```suggestion
#. Open the nRF Device Manager app on your mobile phone, and observe the device now advertising itself as *SUIT Recovery*.
```

> #. Recover the application by performing update in nRF Device Manager app in the same way as described in the ``smp_transfer`` sample documentation, and by using :file:`root.suit` file from the previous build.

Reviewer: ```suggestion
#. Recover the application using the nRF Device Manager app to perform an update, following the process outlined in the :ref:`nrf54h_suit_sample` sample documentation. 
   Use the :file:`root.suit` file generated from the previous build.
```

> #. After the update is complete, the device will reset and start the application

Reviewer: ```suggestion
#. After the update completes, the device will reset and launch the application.
```

> #. Open the nRF Device Manager app on your mobile phone, and observe the device advertising now as "SUIT SMP Transfer".

Reviewer: ```suggestion
#. Open the nRF Device Manager app on your mobile phone and observe that the device is now advertising as *SUIT SMP Transfer*.
```

|test_sample|

#. |connect_kit|
#. Corrupt the currently running main application.
   This can be done by overwriting the beginning of the application partition:

.. code-block:: console

   nrfutil device x-write --address 0xE0A5000 --value 0xFFFF --serial-port COM7

#. Reset the device to enter the recovery path:

.. code-block:: console

   nrfutil device reset --serial-port COM7

#. Open the nRF Device Manager app on your mobile phone, and observe the device now advertising itself as *SUIT Recovery*.
#. Recover the application using the nRF Device Manager app to perform an update, following the process outlined in the :ref:`nrf54h_suit_sample` sample documentation.
   Use the :file:`root.suit` file generated from the previous build.
#. After the update completes, the device will reset and launch the application.

> #. Open the nRF Device Manager app on your mobile phone and observe that the device is now advertising as *SUIT SMP Transfer*.

Reviewer 2: Shouldn't it be:

```suggestion
#. Open the nRF Device Manager app on your mobile phone and observe that the device is now advertising as *SUIT SMP Sample*.
```
